modules/tinymce/src/core/main/ts/mode/Disabled.ts

```diff
@@ -99,6 +104,10 @@ const processDisabledEvents = (editor: Editor, e: Event): void => {
         window.open(href, '_blank', 'rel=noopener noreferrer,menubar=yes,toolbar=yes,location=yes,status=yes,resizable=yes,scrollbars=yes');
       }
     });
+
+    if (editor.hasPlugin('accordion') && hasAccordion(editor, elm)) {
+      e.preventDefault();
+    }
```

Comment:
Why even have the plugin check? The toggle is happening at the DOM level, so the autoresize bug will happen with or without the plungin.

Also, this needs to be better combined with the anchor click code
- [ ] remove hasPlugin
- [ ] Don't run this check if the click was handled by the anchor href code